### PR TITLE
[wip] Stickers

### PR DESCRIPTION
**Pull request by Jikstra**
stickers are here

- Implements a ui driven sticker feature. For testing you can send any telegram sticker pack to the @Stickerdownloadbot, download the png version and unpack it's *.png files into ~/.config/DeltaChat/<yourid>/stickers/<somepackname>.
- Applies a new and more sane folder structure to src/renderer and scss (one more part of moving styled components to sass)

### CODE REVIEW

**Comment by Simon-Laux** [comment]
Can you add a sample / default sticker set? or upload it here for testing

File: src/renderer/components/Composer.js.orig [hunk 1]
```diff
@@ -0,0 +1,365 @@
+import React, { useState, useRef } from 'react'
```
Comment (link2xt) [inline src/renderer/components/Composer.js.orig]: Remove this .orig file

File: src/renderer/components/ChatView.js.orig [hunk 2]
```diff
@@ -0,0 +1,306 @@
+import React, { useRef, useState, useEffect, useContext } from 'react'
```
Comment (link2xt) [inline src/renderer/components/ChatView.js.orig]: And this .orig file

**Comment by Jikstra** [comment]
@Simon-Laux i don't really have a sticker pack that i'm fine with including as a sample pack. But we definetly need one. @kermoshina @jankass can you maybe help us on this?

File: src/renderer/ScreenController.js [hunk 3]
```diff
@@ -3,16 +3,16 @@ const React = require('react')
 const { ipcRenderer } = require('electron')
 
 const ScreenContext = require('./contexts/ScreenContext')
-const UnblockContacts = require('./components/UnblockContacts')
+const UnblockContacts = require('./components/UnblockContactsScreen')
 const LoginScreen = require('./components/LoginScreen').default
-const SplittedChatListAndView = require('./components/SplittedChatListAndView')
+const MainScreen = require('./components/MainScreen')
```
Comment (ralphtheninja) [inline src/renderer/ScreenController.js]: :+1:

**Comment by hpk42** [comment]
maybe also ask @adbenitez who might some ready ;)

**Comment by Simon-Laux** [comment]
I would like to attempt to draw one, so a working example from which I can read the required structure would be appreciated. Do you already have specs for this in the interface or spec repo?

**Comment by Jikstra** [comment]
> I would like to attempt to draw one, so a working example from which I can read the required structure would be appreciated. Do you already have specs for this in the interface or spec repo?

There are no specs on the image size or anything yet.

**Comment by link2xt** [comment]
> I would like to attempt to draw one, so a working example from which I can read the required structure would be appreciated. Do you already have specs for this in the interface or spec repo?

Draw 512x512px PNGs, that is default for telegram.

**Comment by Jikstra** [comment]
Closing in favor of reworked history/split prs. Please do not delete this branch if things go wrong.

**Pull request closed without merging**